core/block.ts

```diff
@@ -962,16 +962,42 @@ export class Block {
   }
 
   /**
-   * @returns True if this block is a value block with a single editable field.
+   * @returns True if this block is a value block with a full block field.
+   * @param mustBeEditable Whether the evaluated field must be editable.
```

Comment:
Nit: Add docs for the `mustBeFullBlock` arg.

Reply:
Done.